fsspec/implementations/cached.py

```diff
@@ -650,7 +651,13 @@ def _open(self, path, mode="rb", **kwargs):
         path = self._strip_protocol(path)
         if "r" not in mode:
             fn = self._make_local_details(path)
-            return LocalTempFile(self, path, mode=mode, fn=fn)
+            user_specified_kwargs = {
+                k: v
+                for k, v in kwargs.items()
+                # those kwargs were added by open(), we don't want them
+                if k not in ["autocommit", "block_size", "cache_options"]
```

Comment:
I wonder if there's any way to guarantee that this list doesn't go stale.

Reply:
No idea as of now. At least, the test should catch addition of new kwargs (it caught it when I forgot to add those three), but it doesn't handle removal.

Regardless, I'd like to find a better way if there is one.